src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -154,7 +154,7 @@ const DebugInformationContainer = connect(
   mapStateToProps,
   null,
   null,
-  { withRef: true }
+  { forwardRef: true }
```

Comment:
Do you have any links that document this change?

Reply:
https://github.com/reduxjs/react-redux/releases/tag/v6.0.0

> Breaking Changes
The withRef option to connect has been replaced with forwardRef. If {forwardRef : true} has been passed to connect, adding a ref to the connected wrapper component will actually return the instance of the wrapped component.